nin/frontend/app/scripts/controllers/bottom.js

```diff
@@ -47,14 +47,16 @@
       $scope.loopEnd = null;
       $scope.loopActive = false;
       commands.on('setCuePoint', function() {
+        var currentBEAN = BEAN_FOR_FRAME($scope.currentFrame);
+        var currentQuantizedFrame = FRAME_FOR_BEAN(currentBEAN - currentBEAN % PROJECT.music.subdivision);
```

Comment:
Yeah, I tried creating it last week, but couldn't get the looping smooth enough with the HTML5 audio element.